takwimu/takwimu_ui/src/components/FeaturedAnalysis/index.js

```diff
@@ -34,32 +34,48 @@ class FeaturedAnalysis extends React.Component {
   render() {
     const {
       classes,
-      takwimu: { featured_analysis: featuredAnalysis }
+      takwimu: {
+        page: {
+          featured_analysis: { value: featuredAnalysis }
+        }
+      }
     } = this.props;
+    if (!featuredAnalysis) {
+      return null;
+    }
+
+    const {
+      title,
+      featured_analyses: featuredAnalyses,
+      read_analysis_link_label: readAnalysisTitle,
+      view_profile_link_label: viewProfileTitle
+    } = featuredAnalysis;
     const { current } = this.state;
 
     return (
-      <Section title="Featured Analysis" variant="h2">
+      <Section title={title} variant="h2">
         <Grid
           container
           justify="flex-start"
           alignItems="stretch"
           className={classes.root}
         >
-          {featuredAnalysis && featuredAnalysis.length ? (
+          {featuredAnalyses && featuredAnalyses.length > 0 && (
             <React.Fragment>
               <CurrentAnalysis
-                content={featuredAnalysis[current].value}
+                content={featuredAnalyses[current]}
                 classes={{ content: classes.content }}
+                readAnalysisTitle={readAnalysisTitle}
+                viewProfileTitle={viewProfileTitle}
               />
               <AnalysisList
-                content={featuredAnalysis}
+                content={featuredAnalyses}
```

Comment:
Typo `Analyses` -> `Analysis`

Reply:
Not a typo, plural form: https://writingexplained.org/analysis-vs-analyses-plural

Reply:
😮